### PR TITLE
adds objects for TB-3302

### PR DESCRIPTION
**Pull request by frankpepermans**
### What 🕵️ 🔍

- See [TB-3302](https://github.com/xaynetwork/xayn_discovery_app/pull/174) in the main app
- Adds the necessary xayn_design updates to support it

----------

### References 📝 🔗

- [TB-3302](https://xainag.atlassian.net/browse/TB-3302)

----------

### CODE REVIEW

File: lib/src/linden/xstyles.dart [hunk 1]
```diff
@@ -71,6 +71,13 @@ class XStyles {
         height: 1.23,
       );
 
+  TextStyle? get appLinkText => appBodyText?.copyWith(
+        fontSize: 10,
+        fontWeight: FontWeight.w700,
```
Comment (pro100svitlo) [inline lib/src/linden/xstyles.dart:76]: 💅 we have `_weightBold`

File: lib/src/linden/xstyles.dart [hunk 2]
```diff
@@ -320,6 +327,27 @@ class XStyles {
         height: 1.6,
       );
 
+  TextStyle? get subscriptionModalTitle => appBodyText?.copyWith(
+        fontSize: 18.0,
+        fontWeight: FontWeight.w400,
```
Comment (pro100svitlo) [inline lib/src/linden/xstyles.dart:332]: 💅 and for those `_weightRegular`

File: lib/src/widget/settings/widget/settings_section.dart [hunk 3]
```diff
@@ -101,7 +101,7 @@ class SettingsSection extends StatelessWidget {
     );
   }
 
-  Widget _buildTitle(Linden linden) => Text(
+  Widget _buildTitle({required String title, required Linden linden}) => Text(
```
Comment (pro100svitlo) [inline lib/src/widget/settings/widget/settings_section.dart:104]: 💅 comma :)

File: lib/src/linden/xstyles.dart (hunk 2 above)
Comment (frankpepermans) [reply lib/src/linden/xstyles.dart:332]: Likewise, changing

File: lib/src/widget/settings/widget/settings_section.dart (hunk 3 above)
Comment (frankpepermans) [reply lib/src/widget/settings/widget/settings_section.dart:104]: Yar

File: lib/src/widget/settings/widget/settings_section.dart [hunk 4]
```diff
@@ -7,7 +7,7 @@ const _defaultAlignment = CrossAxisAlignment.start;
 const _defaultTopPadding = 24.0;
 
 class SettingsSection extends StatelessWidget {
-  final String title;
+  final String? title;
```
[the remark this answers is not in the document]
Comment (frankpepermans) [reply lib/src/widget/settings/widget/settings_section.dart:10]: So [here](https://www.figma.com/file/hGAst3K9ZQIHmQoCNjbgaZ/Discovery-App-%7C-Design?node-id=1019%3A2409), it's without a title, or at least now the common title

**Review by pro100svitlo — APPROVED** [review]

**Pull request merged**